lib/interfaces/gql-module-options.interface.ts

```diff
@@ -44,8 +57,7 @@ export interface GqlModuleOptions
   executorFactory?: (
     schema: GraphQLSchema,
   ) => GraphQLExecutor | Promise<GraphQLExecutor>;
-  installSubscriptionHandlers?: boolean;
-  subscriptions?: GraphQLWsSubscriptionsConfig | boolean;
+  subscriptions?: SubscriptionConfig;
```

Comment:
So the goal with preserving the `installSubscriptionHandlers` option was to remain backward compatible and avoid breaking changes. Basically, whenever someone sets `installSubscriptionHandlers` to `true`, we'd fall back to the default configuration (`subscription-transport-ws`). Would love your thoughts on that 🙌

Reply:
Hi @kamilmysliwiec, I updated the PR, do you thing something like this could work? Setting `installSubscriptionHandlers` enables `subscription-transport-ws` without any specific configuration. I imagine this is useful for subscriptions that don't need any authorization nor additional logic.

Reply:
Sounds great @asmeikal! Thank you